packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.tsx

```diff
@@ -1,70 +1,29 @@
 "use client";
 
+import { faGithub, faDiscord } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { OAuthConnection } from "@thunderstore/dapper/types";
+
 import styles from "./Connections.module.css";
-import { SettingItem } from "../../../SettingItem/SettingItem";
-import { OAuthConnection, UserSettings } from "@thunderstore/dapper/types";
 import { PrivacyPolicyLink } from "../../../Links/Links";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faGithub, faDiscord } from "@fortawesome/free-brands-svg-icons";
-import { useState } from "react";
-import { Switch } from "../../../..";
+import { SettingItem } from "../../../SettingItem/SettingItem";
+import { Switch } from "../../../Switch/Switch";
 
-export interface ConnectionsProps {
-  userData: UserSettings;
-}
+const PROVIDERS = [
+  { name: "Discord", icon: faDiscord },
+  { name: "GitHub", icon: faGithub },
+  { name: "Overwolf", icon: faGithub }, // TODO: icon
```

Comment:
We have `OverwolfLogo` in `packages\cyberstorm\src\svg\svg.tsx`

Reply:
Yeah and no icon wrapper so I'll consider that outside the scope of the PR.

Reply:
I have a open PR for adding wrapper for all the Icons, so mentally I'm not going accept this as a answer. But I'm still going to accept the review and add it myself 👍